transactions/update-contract/2023/may-3-rewards/README.md

```diff
@@ -0,0 +1,38 @@
+# Update FlowIDTableStaking Contract
+
+> May 3, 2023
+
+Upgrade to the `FlowEpoch` contract as per these pull requests:
+
+- https://github.com/onflow/flow-core-contracts/pull/352
+- https://github.com/onflow/flow-core-contracts/pull/365
+
+These include changes to enable automated staking rewards.
+
+## Transaction
+
+[upgrade_and_enable_rewards.cdc](./upgrade_and_enable_rewards.cdc)
+
+Used this to generate args:
+
+`cat "./FlowEpoch.cdc" | xxd -p | tr -d '\n'`
+
+Verified using:
+```
+$ cat arguments-update-contract-FlowEpoch-mainnet.json | jq '.[1] | .value' | xxd -r -p > /tmp/temp.txt
+$ diff /tmp/temp.txt FlowEpoch.cdc
+(Should produce no difference)
+```
```

Comment:
👍  To double-check, I also verified against the current master contract version:
```
 % cat arguments-update-contract-FlowEpoch-mainnet.json | jq '.[1] | .value' | xxd -r -p > /tmp/temp.txt
dapperlabs@J-Mac-DL may-3-rewards % diff /tmp/temp.txt /Users/dapperlabs/dev/src/github.com/onflow/flow-core-contracts/contracts/epochs/FlowEpoch.cdc
1,6c1,6
< import FungibleToken from 0xf233dcee88fe0abe
< import FlowToken from 0x1654653399040a61
< import FlowIDTableStaking from 0x8624b52f9ddcd04a
< import FlowClusterQC from 0x8624b52f9ddcd04a
< import FlowDKG from 0x8624b52f9ddcd04a
< import FlowFees from 0xf919ee77447b7497
---
> import FungibleToken from 0xFUNGIBLETOKENADDRESS
> import FlowToken from 0xFLOWTOKENADDRESS
> import FlowIDTableStaking from 0xFLOWIDTABLESTAKINGADDRESS
> import FlowClusterQC from 0xQCADDRESS
> import FlowDKG from 0xDKGADDRESS
> import FlowFees from 0xFLOWFEESADDRESS
```

Reply:
thanks for double checking!